src/extensionVariables.ts

```diff
@@ -59,8 +59,6 @@ export namespace ext {
     // Container runtime related items
     export let runtimeManager: ContainerRuntimeManager;
     export let orchestratorManager: OrchestratorRuntimeManager;
-    export const runWithDefaultShell = runWithDefaultShellInternal;
-    export const streamWithDefaultShell = streamWithDefaultShellInternal;
-    export const runOrchestratorWithDefaultShell = runOrchestratorWithDefaultShellInternal;
```

Comment:
The orchestrator ones weren't actually being used, so I opted to just remove them. If we need them in the future they can be added back.